litemapy/schematic.py

```diff
@@ -34,8 +34,8 @@ def __init__(self,
         self.__compute_enclosure()
         if regions is not None and len(regions) > 0:
             self.__regions.update(regions)
-        self.mc_version = mc_version
-        self.lm_version = lm_version
+        self.__mc_version = mc_version
+        self.__lm_version = lm_version
```

Comment:
To be honest, I haven't looked into Python's philosophy on protected variables too much. Especially because - as I understand it - it can be circumvented anyway(?). So I went for consistency here. I don't have strong feelings either way and would leave the decision up to you.

Reply:
Fields and methods (they are in fact the same thing) that start and do not end with double underscores are indeed private in python:
```python
>>> class TestClass:
...  def __init__(self):
...   self.__test_field = "Hello"
...  
...  def print_test_field(self):
...   print(self.__test_field)
... 
>>> t = TestClass()    
>>> t.__test_field
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'TestClass' object has no attribute '__test_field'. Did you mean: 'print_test_field'?
>>> t.print_test_field()
Hello
```
The intention behind making them private and creating accessors using decorators in litemapy was to either make some fields read only or to do some type checking/transformation.
It makes no difference for calling code whether a field is indeed one or if it's just a pair of accessors accessing an underlying private field, so if there is no real benefit to protecting the field I rather not do it to avoid adding code that doesn't make any difference in the end.

Reply:
Sounds good. I removed the protection for the version numbers